ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -122,6 +122,7 @@ message DamlSubmitterInfo {
   string command_id = 2;
   string application_id = 3;
   google.protobuf.Timestamp maximum_record_time = 4;
+  google.protobuf.Timestamp ttl = 5;
```

Comment:
Shouldn't this be `expiry_time` instead? I.e., `ttl` means the lifetime of a command in relative terms in places (e.g., in the configuration) and in other places it means the time at which the command is going to expire. We should use consistent naming.

Reply:
Good point, the `TTL` name is not very good, I'll rename it everywhere.

Reply:
:thumbsup: